resolwe/toolkit/docker_images/base/Dockerfile.fedora-31

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/fedora:31
```

Comment:
Also, remove base image for Fedora 26 and migrate all images to this one.

https://github.com/genialis/resolwe/blob/master/resolwe/toolkit/docker_images/upload-tab-file/Dockerfile
https://github.com/genialis/resolwe/blob/master/resolwe/toolkit/docker_images/archiver/Dockerfile